.github/workflows/pr.yml

```diff
@@ -12,7 +12,7 @@ jobs:
         uses: ammaraskar/sphinx-action@master
         with:
           docs-folder: "docs/"
-          pre-build-command: "pip install pipenv && pipenv install && pipenv shell"
+          pre-build-command: "pip install sphinx-rtd-theme && pip install pipenv && pipenv install && pipenv shell"
```

Comment:
This isn't needed.  the end of the pip install will take care of installation of the package if it's in the Pipfile and Pipfile.lock.

Reply:
Hey @markpbaggett I did try to remove these and yet it still fails the _build_ workflow run. See https://github.com/UTKcataloging/mods_to_rdf/pull/53/commits/1bba996d2e56277ad806bfc8cdbc74c1bcb6c156